src/SDKs/Monitor/Monitor.Tests/Monitor.Tests.csproj

```diff
@@ -22,7 +22,7 @@
 
 	<ItemGroup Condition=" '$(TargetFramework)' == 'net452' ">
 		<Reference Include="System.Net.Http" />
-		<PackageReference Include="Newtonsoft.Json" Version="6.0.8" />
+    <PackageReference Include="Newtonsoft.Json" Version="6.0.8" />
```

Comment:
@shriku11 if the only change you made is to delete indentation, can you undo this, so that there is no history of this change plus the indentation is back to normal.

Reply:
Fixed the indentation.